PyPDF2/_utils.py

```diff
@@ -181,7 +178,7 @@ def read_previous_line(stream: StreamType) -> bytes:
         if not found_crlf:
             # We haven't found our first CR/LF yet.
             # Read off characters until we hit one.
-            while idx >= 0 and block[idx] not in CRLF:
+            while idx >= 0 and block[idx] not in b"\r\n":

```

Comment:
Curious why this change was suggested? Just to avoid polluting what gets imported from doing a `import * from ._utils`?

Reply:
Apparently using global variables has a slight performance penalty.

Typically I would have preferred readability over this minor performance hit, but in this case I think it is similar readable

Reply:
Yeah, makes sense I guess in how the scope lookup for variables works, though this feels like micro-optimization territory that only matters if you're running the code millions upon millions times you'd notice a slight speedup.

Interesting nevertheless as I've never used `perflint` before. 🤷

Reply:
I was just listening to some PyCon talks where it was presented :-) The author also said that it might give several false-positives, so one has to use it with a bit of caution.

Reply:
> this feels like micro-optimization territory that only matters if you're running the code millions upon millions times

I definitely agree! If it would have made the code less readable, I would not have applied it.

I've just looked at some call graphs (e.g. https://github.com/py-pdf/PyPDF2/issues/984 ) and we do run some functions quite a lot of times for small PDF documents. Those functions look fine to me though.